.ci-dockerfiles/ubuntu/README.md

```diff
@@ -0,0 +1,33 @@
+# Build image
+
+```bash
+docker build -t ponylang/ponyc-ci:ubuntu .
+```
+
+# Run image to test
+
+Will get you a bash shell in the image to try cloning Pony into where you can test a build to make sure everything will work before pushing:
+
+```bash
+docker run --name ponyc-ci-ubuntu --user pony --rm -i -t ponylang/ponyc-ci:ubuntu bash
+```
+
+# Run CircleCI jobs locally
+
+Use the [CircleCI CLI](https://circleci.com/docs/2.0/local-cli/) to run the CI job using this image
+from the ponyc project root:
+
+```bash
+circleci build --job ubuntu-debug
+circleci build --job ubuntu-release
+circleci build --job ubuntu-llvm-600-debug
+circleci build --job ubuntu-llvm-600-release
```

Comment:
the llvm-600 should be removed. that doesnt exist.

Reply:
See first reply [above](https://github.com/ponylang/ponyc/pull/2663#discussion_r191079706)